graph/Cargo.toml

```diff
@@ -65,3 +65,4 @@ web3 = { git = "https://github.com/graphprotocol/rust-web3", branch = "master" }
 test-store = { path = "../store/test-store" }
 maplit = "1.0.2"
 structopt = { version = "0.3" }
+graph-chain-ethereum = { path = "../chain/ethereum" }
```

Comment:
I've been trying very hard to avoid this; totally fine as a temporary thing, but should have an `ETHDEP` comment with it that it needs to be removed before we can consider the refactor finished.

Reply:
This is a `dev-dependencies`. To test things that live in `graph` we might need _some_ concrete implementation of the traits, I don't see harm in testing with Ethereum. This of course could never be a regular dependency since those cannot be circular.